buildSrc/src/main/kotlin/KotlinCommunity.kt

```diff
@@ -37,4 +37,19 @@ fun addDevRepositoryIfEnabled(repositoryHandler: RepositoryHandler, project: Pro
     repositoryHandler.maven {
         url = URI.create(devRepoUrl)
     }
+}
+
+/**
+ * Should be used for running against non-released Kotlin compiler on a system test level.
+ *
+ * @return a Kotlin API version parametrized from command line or gradle.properties, null otherwise
+ */
+fun getOverriddenKotlinNativeVersion(project: Project): String? {
+    val nativeVersion = project.rootProject.properties["kotlin.native.version"] as? String
+    return if (!nativeVersion.isNullOrBlank()) {
+        project.logger.info("""Configured Kotlin Native distribution version: '$nativeVersion' for project $${project.name}""")
```

Comment:
`$${project.name}` seems to be a typo:
> Configured Kotlin Native distribution version: '2.1.255-SNAPSHOT' for project **$integration**

Reply:
```suggestion
        project.logger.info("Configured Kotlin Native distribution version: '$nativeVersion' for project ${project.name}")
```

nitpick: it doesn't need to be a raw string either

Reply:
Removed the redundant dollar sign and preserved the triple double quotes for consistency with other log strings in this file.